src/main/java/FindTheMinimum.java

```diff
@@ -1,10 +1,9 @@
-package main;
```

Comment:
Since it's not possible to write comments for lines not related to code I'll write them here:

We can rewrite 
> getTwoMinimum(int a, int b)
several ways to make it more clear and short:

`public static int getTwoMinimum(int a, int b) {
    if (a > b) {
        return b;
    }
    return a;
}`

or even in one line:
`public static int getTwoMinimum(int a, int b) {
    return a > b ? b : a;
}`

But your solution is fine.

Reply:
:) copy that, thanks!